packages/start/src/client/serialization.tsx

```diff
@@ -101,9 +101,9 @@ export function afterHydrate({ router }: { router: AnyRouter }) {
         Object.entries(extracted).forEach(([_, ex]: any) => {
           if (ex.value instanceof Promise) {
             const og = ex.value
-            ex.value = og.then((data: any) =>
-              router.options.transformer.parse(data),
-            )
+            ex.value = og.then((data: any) => {
+              return data
```

Comment:
When it gets here, it's already in Javascript form (ie: `Object`, `Array`, ...) and not just a string.